pages/agent/v2/github_ssh_keys.md

```diff
@@ -27,5 +26,5 @@ An alternative method of providing access to your repositories is to use deploy
 
 To setup GitHub deploy keys with the Buildkite Agent, you'll need to do the following for each repository:
 
-* On your agent machine, generate a key as per the [agent SSH keys documentation](/docs/agent/v2/ssh- keys#creating-multiple-ssh-keys)
-* In GitHub, copy the key into the repository's “Deploy keys” settings
+* On your agent machine, generate a key as per the [agent SSH keys documentation](/docs/agent/v2/ssh-keys#creating-multiple-ssh-keys)
+* In GitHub, copy the key into the repository's "Deploy keys" settings
```

Comment:
PS after going through the whole thing, I think this was intentional. This was the first time I saw the change.

Reply:
This is intentional, using simple (rather than fancy directional) double-quotation marks is a safer bet if we decide to change do something with the site's formatting (e.g. CSS styles) later down the track.